lib/geordi/db_cleaner.rb

```diff
@@ -103,40 +112,43 @@ def edit_whitelist(dbtype)
     end
 
     def decide_mysql_command(extra_flags)
-      cmd = 'sudo mysql'
+      cmd = @system_user ?'sudo mysql' : 'mysql'
```

Comment:
Missing a space here
```suggestion
      cmd = @system_user ? 'sudo mysql' : 'mysql'
```